update_compilers/install_compilers.sh

```diff
@@ -413,6 +413,15 @@ if [ ! -d "libs/boost_1_64_0" ]; then
     popd
     rm -rf /tmp/boost
 fi
+
+# nvcc
```

Comment:
np. can rework that at some stage too. I'm also very limited in the cuda domain knowledge too. I did enough to get some ASM out of it.